mongo/entrypoint.sh

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env bash
 # set -e
 
-BASH_PATH=$(which bash)
+BASH_PATH=$(command -v bash)
```

Comment:
I created the variable to avoid repeating `$(command -v)`.
Also I changed `which` for `command -v` because my IDE started to complain about `which` was not standard (https://github.com/koalaman/shellcheck/wiki/SC2230). 
I read a little bit about it and it seems it's true. I also asked @rroux who confirmed it. 

I hear you, though. Should I replace `BASH_PATH` with `/bin/bash` then?

Reply:
We are not alone: 
https://unix.stackexchange.com/questions/43499/difference-between-echo-shell-and-which-bash

Reply:
OK, so `SHELL` is bad :blush:
Let's drop `BASH_PATH` in favor of just `bash` (without an absolute path). If it fails, you can make fun of me on the next video call :smile: